src/humanize/time.py

```diff
@@ -590,7 +590,8 @@ def precisedelta(
     for unit, fmt in zip(reversed(Unit), fmts):
         singular_txt, plural_txt, fmt_value = fmt
         if fmt_value > 0 or (not texts and unit == min_unit):
-            fmt_txt = _ngettext(singular_txt, plural_txt, fmt_value)
+            _fmt_value = 2 if fmt_value > 1 and fmt_value < 2 else int(fmt_value)
+            fmt_txt = _ngettext(singular_txt, plural_txt, _fmt_value)
```

Comment:
This looks good.

And we can simplify a chained comparison:

```suggestion
            _fmt_value = 2 if 1 < fmt_value < 2 else int(fmt_value)
            fmt_txt = _ngettext(singular_txt, plural_txt, _fmt_value)
```

Reply:
Oh okay, thanks!